cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -64,6 +64,14 @@ spec:
       labels:
         k8s-app: kube-proxy
     spec:
+      initContainers:
+      - name: touch-lock
+        image: {{ .ImageRepository }}/busybox-{{ .Arch }}
```

Comment:
Let me test with https://github.com/kubernetes/kubernetes/pull/47212 quick and see if that will work as an interim